src/Col.js

```diff
@@ -78,8 +79,7 @@ const Col = (props) => {
 
       colClasses.push(mapToCssModules(classNames({
         [colClass]: columnProp.size || columnProp.size === '',
-        [`push${colSizeInterfix}${columnProp.push}`]: columnProp.push || columnProp.push === 0,
-        [`pull${colSizeInterfix}${columnProp.pull}`]: columnProp.pull || columnProp.pull === 0,
+        [`order${colSizeInterfix}${columnProp.order}`]: columnProp.order || columnProp.order === 0,
```

Comment:
I don't think `0` should trigger a class to be added (`order-*-0`). previously, push and pull could be "reset" for a breakpoint (where `0` came into play), but it doesn't look like order has that ability.
This is just a note as I am going to leave it in as the end user should never use `0` his/her self. Also, bootstrap may add `0` in the future to allow a reset